tensorflow/lite/micro/tools/ci_build/test_all.sh

```diff
@@ -55,4 +55,7 @@ tensorflow/lite/micro/tools/ci_build/test_arduino.sh
 echo "Running cortex_m_generic tests at `date`"
 tensorflow/lite/micro/tools/ci_build/test_cortex_m_generic.sh
 
+echo "Running ESP32 tests at `date`"
+tensorflow/lite/micro/tools/ci_build/test_esp32.sh
+
```

Comment:
let's remove this from the current pull request -- this will trigger the ESP script on every pull request and we want to make sure that the test_esp32.sh script is stable and doesn't take too much time.

Once the test_esp32.sh is merged, I'll enable it as part of a build that runs every 4 hours and add a build badge [here](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/micro#continuous-buils-status).

Reply:
@advaitjain is it okay to do force push in this situation? 

Adding extra fix commit seems unreasonable.

Reply:
Addressed with an extra commit for now.

Reply:
Thanks! Please always add an extra commit, even for seemingly small changes.